smoke_tests/tests/authenticated/landingzone_definitions_tests.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from tests.smoke_test_case import SmokeTestCase
+
+"""Validates landingzone definitions endpoint"""
+
+
+class LandingZoneDefinitionsTests(SmokeTestCase):
+    CROMWELL_LZ_DEFINITION_NAME = "CromwellBaseResourcesFactory"
+
+    @staticmethod
+    def status_url() -> str:
+        return SmokeTestCase.build_lz_url("/api/landingzones/definitions/v1/azure")
+
+    """Validates that definitions endpoint return 401 when no token provided"""
```

Comment:
The placement of this docstring should be inside the method like so https://peps.python.org/pep-0257/#one-line-docstrings, curious if `black` will pick this up.